### ISSUE TITLE
Immortal Unburried

### ISSUE THREAD

**Issue by Chinthor** [adam9899/Issues#729]
There have been many comments on how OP the Undertaker's Unburied minions are. Some posts are linked to from here. Unfortunately, all issues posted to Fish's Undead Rising were recently deleted. A new issue concerning missing issues has been posted.

I don't know how hard these are supposed to be to kill, but I'm frequently seeing immortal Unburied. Upgraded crossbow bolts that do ~25 dmg to normal targets and 30+ vs undead do no damage at all. Unloaded over 30 bolts into it. Same for mele with 15 dmg demonmetal sword. No effect. To kill one such Unburied, I went into creative mode and smacked it with a Kikoku. The god-killing, void-damage dealing, creative player killer. This did only 2 damage and took over 40 strikes because it also had regen.

I'm ok with these things and the Undertaker being difficult, tanky, and dealing 1.8 full health bars of armor piercing damage in one touch. But not if they're immortal. Especially these guys, as they pop up through walls and my basement.

Is there a way to fix this without nerfing them all the way back to original stats?

**Comment by JoeGribble** [comment]
I agree with this issue - I'm trying to do MC Eternal with a Minecolonies focus, and it's extremely annoying in the early game without access to hard-hitting tools having an Undertaker spawn in with 3500 Hearts, totally wrecking you whenever you get close to your colony. It's extremely skewed towards late-game, and makes the pack unappealing.
